desktop/cmp/dash/canvas/DashCanvas.ts

```diff
@@ -115,6 +115,8 @@ const emptyContainerOverlay = hoistCmp.factory<DashCanvasModel>(({model}) => {
 });
```

Comment:
is there an analog here DashContainer?

Reply:
@lbwexler are you asking about an analog emptyContainerOverlay?  or an analog e.preventDefault()?

Reply:
@lbwexler if you are asking about an analog e.preventDefault(), I think that line 464 `return false;` here:
https://github.com/xh/hoist-react/blob/develop/desktop/cmp/dash/container/DashContainerModel.ts#L464 is the analog.   I tested Toolbox's dashcontainer demo and found that it did not have this problem.